solution4.py

```diff
@@ -0,0 +1,20 @@
+import csv
+data = [
+    ["2", "om", "4000"],
+    ["5", "sai", "4200"]
+]
+
+header = ["eid", "ename", "esalary"]
+with open("salary.csv", mode="w", newline="") as file:
+    writer = csv.writer(file)
+    writer.writerow(header)

```

Comment:
what is the different between "writerow" and "writerows" ?

Reply:
writterow() in This method writes a single row at a time. and writterows in this  method writes a multiple rows at a time.